co-authors-plus.php

```diff
@@ -1610,13 +1610,22 @@ public function clear_cache_on_terms_set( $object_id, $terms, $tt_ids, $taxonomy
 
 	/**
 	 * Filter of the header of author archive pages to correctly display author.
+	 *
+	 * @param $title
+	 *
+	 * @return string
```

Comment:
This is just a picky thing, but if you get the time, would you mind completing the inline doc following the standard `$var_name var_type short_var_description` format? Since you have added them, which is great, we may also complete them for future reference :)